### PR TITLE
[WIP] Order placed in channel currency

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

This PR should be based on #6791 and #6794, but I have extracted changes to make diff more readable.

### CODE REVIEW

File: src/Sylius/Behat/Context/Setup/CurrencyContext.php [hunk 1]
```diff
@@ -126,16 +126,12 @@ public function theStoreHasCurrencyWithExchangeRate($currencyCode, $exchangeRate
     }
 
     /**
-     * @Given /^(that channel) allows to shop using the "([^"]+)" currency$/
+     * @Given /^(that channel) allows to shop using the "([^"]+)" currency(?: also|)$/
```
Comment (pamil) [inline src/Sylius/Behat/Context/Setup/CurrencyContext.php]: "also" should be put just after "that channel"

File: src/Sylius/Behat/Context/Setup/CurrencyContext.php [hunk 2]
```diff
@@ -126,16 +126,12 @@ public function theStoreHasCurrencyWithExchangeRate($currencyCode, $exchangeRate
...
      * @Given /^(that channel) allows to shop using "([^"]+)" and "([^"]+)" currencies$/
      * @Given /^(that channel) allows to shop using "([^"]+)", "([^"]+)" and "([^"]+)" currencies$/
      */
     public function thatChannelAllowsToShopUsingAndCurrencies(ChannelInterface $channel, ...$currenciesCodes)
     {
-        foreach ($channel->getCurrencies() as $currency) {
-            $channel->removeCurrency($currency);
```
Comment (pamil) [inline src/Sylius/Behat/Context/Setup/CurrencyContext.php]: Why?

File: src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php [hunk 3]
```diff
@@ -836,4 +836,18 @@ public function theAdministratorShouldKnowAboutIPAddressOfThisOrderMadeBy(AdminU
             );
         });
     }
+    /**
+     * @Then /^(the administrator) should see that (order placed by "customer@example.com") has "([^"]+)" currency$/
```
Comment (pamil) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php]: Hardcoded email

File: src/Sylius/Behat/Context/Ui/Shop/CurrencyContext.php [hunk 4]
```diff
@@ -50,6 +50,7 @@ public function __construct(HomePageInterface $homePage, CurrencyStorageInterfac
     /**
      * @When I switch to the :currencyCode currency
      * @When I change my currency to :currencyCode
+     * @When I changed my currency to :currencyCode
```
Comment (pamil) [inline src/Sylius/Behat/Context/Ui/Shop/CurrencyContext.php]: `@Given`

File: src/Sylius/Behat/Context/Setup/CurrencyContext.php (hunk 2 above)
Comment (lchrusciel) [inline src/Sylius/Behat/Context/Setup/CurrencyContext.php]: Why should we remove remove all of previous currencies? Is there any particular reason for that? Anyway, we should probably change logic of getCurrencies

File: features/checkout/placing_order_on_multi_channel/placing_an_order_on_a_multiple_channels_with_same_currency.feature [hunk 5]
```diff
@@ -0,0 +1,33 @@
+@checkout
+Feature: Placing an order on a multiple channels with same currency
```
Comment (GSadee) [inline features/checkout/placing_order_on_multi_channel/placing_an_order_on_a_multiple_channels_with_same_currency.feature]: Without `a` IMO

File: features/checkout/placing_order_on_multi_channel/placing_an_order_on_a_multiple_channels_with_same_currency.feature [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    In order to be sure how much should I pay for my cart
+    As an Administrator
+    I want user to place orders in channel’s base currency
+
+    Background:
+        Given the store operates on a channel named "Los Angeles" in currency "USD"
+        Given the store operates on another channel named "New York" in currency "USD"
```
Comment (GSadee) [inline features/checkout/placing_order_on_multi_channel/placing_an_order_on_a_multiple_channels_with_same_currency.feature]: `And`

**Comment by lchrusciel** [comment]
Closing in favour of #6810

**Pull request closed without merging**